src/packet.rs

```diff
@@ -63,6 +63,19 @@ impl Default for Packet {
     }
 }
 
+pub trait Recycle {
```

Comment:
I would have called this `Recyclable`, but I don't see a precedent for that "<verb>-able" naming convention in Rust, which is too bad. Looks like Rustaceans would name it `Reset`. Thoughts?

Reply:
Reset works for me, too.  I didn't think -able looked right.